pkg/envtest/komega/default.go

```diff
@@ -0,0 +1,85 @@
+package komega
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// defaultK is the Komega used by the package global functions.
+var defaultK = &komega{}
+
+// SetDefaultClient sets the client used by the package global functions.
+func SetDefaultClient(c client.Client) {
```

Comment:
I wonder if this should be just SetClient()

I'm not sure in which way it is a default client.